include/nbl/builtin/hlsl/math/quadrature/gauss_legendre/impl.hlsl

```diff
@@ -15,58 +15,40 @@ namespace math
 {
 namespace quadrature
 {
```

Comment:
by doing this you'll end up with the arrays being in `nbl::hlsl::math::quadrature::nbl::hlsl::math::quadrature::impl`